src/vcpkg/commands.build.cpp

```diff
@@ -1561,17 +1582,25 @@ namespace vcpkg
                        "**Failure logs**\n\n```\n{}\n```\n",
                        paths.get_filesystem().read_contents(build_result.stdoutlog.value_or_exit(VCPKG_LINE_INFO),
                                                             VCPKG_LINE_INFO));
-        result.append(Strings::join("\n", build_result.error_logs, CreateLogDetails{fs}));
 
+        std::string postfix;
         const auto maybe_manifest = paths.get_manifest();
         if (auto manifest = maybe_manifest.get())
         {
-            fmt::format_to(
-                std::back_inserter(result),
-                "**Additional context**\n\n<details><summary>vcpkg.json</summary>\n\n```\n{}\n```\n</details>\n",
-                Json::stringify(manifest->manifest));
+            if (args.get_command() != "x-set-installed" || manifest->manifest.contains("builtin-baseline"))
+            {
+                fmt::format_to(
+                    std::back_inserter(postfix),
+                    "**Additional context**\n\n<details><summary>vcpkg.json</summary>\n\n```\n{}\n```\n</details>\n",
+                    Json::stringify(manifest->manifest));
+            }
         }
 
+        int max_body_size = MAX_ISSUE_SIZE - static_cast<int>(result.size()) - static_cast<int>(postfix.size());
```

Comment:
This needs to be done in `size_t` space not `int` space.

Reply:
But `result.size() + postfix.size()` can be greater than `MAX_ISSUE_SIZE`

Reply:
> But `result.size() + postfix.size()` can be greater than `MAX_ISSUE_SIZE`

Calculating the total size should be done in `size_t` space, and that compared with `MAX_ISSUE_SIZE`. Or alternately make `MAX_ISSUE_SIZE` itself a `size_t`.